arbnode/seq_coordinator.go

```diff
@@ -175,6 +176,9 @@ func NewSeqCoordinator(streamer *TransactionStreamer, sequencer *Sequencer, conf
 	if err != nil {
 		return nil, err
 	}
+	if config.MyUrl == "" {
+		config.MyUrl = INVALID_URL
```

Comment:
Could you explain the reasoning behind this? Why not return an error here? Is the intention to allow the sequencer to operate but not acquire the lockout? If so, I think we should have it disable its own liveliness.

Reply:
disabling own liveliness is a good callout. Will definitely add it.
We need sequencers that have coordinator but no valid URL because they can be used as endpoints - and will forward messages to the right sequencer.